[3.31.0] CI builds fails, command not found: bob

### Description

After running `yarn install` on CI on a  macos 14 machine the install fails with: 

`react-native-screens@npm:3.31.0 [e76ba] couldn't be built successfully (exit code 127, logs can be found here: /private/var/folders/cm/2r_39vcd593436b3_ht4nyqw0000gp/T/xfs-1e614b9e/build.log)`

the log states: 

`command not found: bob`

### Steps to reproduce

1. I'm using yarn 4.1.1
2. use latest version of react-native-screens 3.31.0
3. yarn install fails

### Snack or a link to a repository

no-repo-needed

### Screens version

3.31.0

### React Native version

0.73.7

### Platforms

Android, iOS

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Paper (Old Architecture)

### Build type

Release mode

### Device

Real device

### Device model

_No response_

### Acknowledgements

Yes

Experiencing the same behaviour on Windows and RN@0.74
```
npm ERR! code 1
npm ERR! path C:\Users\XXXX\projects\lahoco-mobile\node_modules\react-native-screens
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c bob build && husky install
npm ERR! 'bob' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
[...]
Process finished with exit code 1
```

@efstathiosntonas 

```
npm ERR! command sh -c bob build && husky install
npm ERR! ℹ Building target commonjs
npm ERR! ℹ Cleaning up previous build at lib/commonjs
npm ERR! ℹ Compiling 61 files in src with babel
npm ERR! ✔ Wrote files to lib/commonjs
npm ERR! ℹ Building target module
npm ERR! ℹ Cleaning up previous build at lib/module
npm ERR! ℹ Compiling 61 files in src with babel
npm ERR! ✔ Wrote files to lib/module
npm ERR! ℹ Building target typescript
npm ERR! ℹ Cleaning up previous build at lib/typescript
npm ERR! ℹ Generating type definitions with tsc
npm ERR! ✖ Couldn't find a tsconfig.json in the project root.
npm ERR! bob build
```

npm ERR! code 127
npm ERR! path node_modules/react-native-screens
npm ERR! command failed
npm ERR! command sh -c bob build && husky install
npm ERR! sh: bob: command not found

--- a/package.json
+++ b/package.json
@@ -14,8 +14,8 @@
     "lint-js": "eslint --ext '.js,.ts,.tsx' --fix src",
     "lint-android": "./android/gradlew -p android spotlessCheck -q",
     "lint": "yarn lint-js && yarn lint-android",
-    "release": "npm login && release-it",
-    "postinstall": "bob build && husky install"
+    "release": "yarn prepare && npm login && release-it",
+    "prepare": "bob build && husky install"
   },
   "main": "lib/commonjs/index",
   "module": "lib/module/index",

